Add list of how we prioritize an issue (#624)
medic/cht-docs: content/en/contribute/_index.md

- [Share your expertise](https://forum.communityhealthtoolkit.org) on our Forum
- Sign up for our [newsletter](https://communityhealthtoolkit.org/contact) to get the latest updates

**Are you a partner wondering how issues are prioritized?**

> Issues that a partner will: 

Reviewer: Referred to as "tickets" below. I tend to prefer this since "issues" is GitHub specific and otherwise sounds like something is a problem. Not that big of a deal though, as long as we're consistent.

```suggestion
Tickets that a partner will: 
```

> There are a number of factors that contribute to prioritization of work. The following represents important factors that are often evaluated together to have the best change for the largest impact if a partner will: 

Reviewer: I think this line was from me in the first place, so here's my correction of my own error:
```suggestion
There are a number of factors that contribute to prioritization of work. The following represents important factors that are often evaluated together to have the best chance for the largest impact if a partner will: 
```

* Upgrade to get the feature/bugfix
* Help validate, design and prototype the feature 
* Help fund the development
* Be blocked by it for a major release or expansion 

> * Has users that are unable to use the CHT or experience it in a way that results in providing bad care

Reviewer: ```suggestion
* Has users that are unable to use the CHT or experience it in a way that results in providing degraded care
```

> Will get prioritized.  Futher, internally, a ticket may be needed to facilitate a release or reduce technical debt.

Reviewer: A spelling correction and suggestion for wording of tech debt.

```suggestion
Will get prioritized.  Further, internally, a ticket may be needed to facilitate a release or perform important upgrades as is important to do in the maintenance of software.
```

Author: The close proximity of the two "important"s in your suggestion feels awkward. How about:

```suggestion
Will get prioritized.  Further, internally, a ticket may be needed to facilitate a release or perform important upgrades as is critical to do in the maintenance of software.
```
